libbeat/logp/logger.go

```diff
@@ -0,0 +1,31 @@
+package logp
```

Comment:
some parts of the pipeline allow for a logger to be passed by interface/type. This allows the more 'expensive' tests to capture the log output in the test context -> correctly group log messages with test output.